java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1171,6 +1171,44 @@ public void zscore(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zdiff(BaseClient client) {
+        String key1 = "{testKey:}" + UUID.randomUUID();
```

Comment:
nit: what's the benefit of adding `"{testKey:}"` here? I think some of Yury's PRs add something like `"{setKey}-1-"` or `"{test}-hll1-"`which adds a bit more specificity to the key but I'm not sure if there's a benefit to adding just "testKey". Not a big deal though, I know other tests already prepend the generic "{key}". Just personally think we should either not prepend the UUID or prepend with the type of the key. Then again, prepending with the type of the key might not be adding much benefit either (unless I'm missing something)

Reply:
{testKey:} if you do not prepend this with multi key commands you will run into the error CrossSlot: Keys in request don't hash to the same slot. Otherwise, you don't need to add it. Doesn't  matter if you use {testKey:} or {setKey}-1- .